frontend/components/text2text/results/Text2TextList.vue

```diff
@@ -350,16 +349,19 @@ export default {
         }
       }
     },
-    async onAnnotate(sentence) {
+    async onValidate() {
       let newS = {
         score: 1,
-        text: sentence,
+        text: this.record.sentenceForAnnotation,
```

Comment:
is sentenceForAnnotation always a properties of record ? if not, manage the undefined case